Beams/MRML/vtkMRMLRTPlanNode.cxx

```diff
@@ -177,16 +156,40 @@ void vtkMRMLRTPlanNode::Copy(vtkMRMLNode *anode)
   this->InvokePendingModifiedEvent();
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLRTPlanNode::CopyContent(vtkMRMLNode *anode, bool deepCopy/*=true*/)
+{
+  MRMLNodeModifyBlocker blocker(this);
+  Superclass::CopyContent(anode, deepCopy);
+
+  vtkMRMLRTPlanNode* node = vtkMRMLRTPlanNode::SafeDownCast(anode);
+  if (!node)
+  {
+    return;
+  }
+
+  this->SetRxDose(node->GetRxDose());

```

Comment:
There are macros for this as well: https://github.com/Slicer/Slicer/blob/master/Libs/MRML/Core/vtkMRMLNodePropertyMacros.h#L370-L451